src/components/common/accordion/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
as i remember import of react no needed more, or gatsby requires it?

Reply:
Yes.